perllib/FixMyStreet/Cobrand/Base.pm

```diff
@@ -53,6 +53,230 @@ sub is_default {
 
 # NB: this Base class is for 'meta' features.  To add base methods for all cobrands,
 # you may want to look at FMS::Cobrand::Default instead!
+=head2 setup_contacts, ensure_bodies
```

Comment:
This should be in Default.pm, not Base, as per the comment just above. Although I'm not sure it makes sense to be in Default either, as it's just for development; could this code live in the `setup-contacts` script, with just the cobrand-specific data in the cobrand file? It doesn't seem like it would be shared with anything.